packages/integration-tests/src/fixtures/static-ip/static-network-config.ts

```diff
@@ -0,0 +1,17 @@
+const dummyStaticNetworkConfig = [
```

Comment:
I would suggest an additional scenario - hostsConfigured that validates the Hosts specific configurations form loaded correctly for an existing cluster (since there were a few bugs in this specific scenario)

Reply:
Thanks, that's a good idea. I'll do that.
We can enhance this suite more in the future too, adding more scenarios (YAML view etc)

Reply:
Hi, I added these scenarios about loading from the InfraEnv to the form view.
One for each sub-step.

https://github.com/openshift-assisted/openshift-assisted-ui/pull/28/commits/67898a8fb5a268eeeb9b99247c1025bfe2275d51